Listado y búsqueda de usuarios-#132

alemunara opened the pull request:
Finalizado el buscador de usuarios y la visualización del perfil de otros usuarios

alemunara committed:
Listado y búsqueda de usuarios-#132

Finalizado el buscador de usuarios y la visualización del perfil de otros usuarios

alemunara committed:
Merge branch 'develop' of https://github.com/youRoom-ISPP/youRoom into LU-#132

fragonlug reviewed (approved):
Se ha validado el funcionamiento del nuevo caso de uso

jorandmol reviewed (changes requested):
He comprobado que funciona todo bien excepto que el usuario no puede votar desde el perfil de otro usuario. Aparecen las estrellas, por lo que tiene posibilidad, pero no se realiza la votación.

Quitaría dicha opción o lo arreglaría, siendo mejor la segunda opción.

alemunara committed:
Valoraciones en perfil de usuarios-#132

Se ha corregido un error de integración con backend para la valoración de publicaciones en perfiles de otros usuarios.

alemunara commented:
Se ha corregido el error identificado al valorar publicaciones

jorandmol reviewed (approved):
He comprobado que se puede valorar desde la vista de usuarios, por lo que apruebo la petición.
